mkdocs.yml

```diff
@@ -1,11 +1,11 @@
 site_name: Ideascube
 pages:
-- [index.md, Home]
-- [install.md, Installation]
-- [config.md, Configuration]
-- [contributing.md, Contributing]
-- [i18n.md, Translations]
-- [release_process.md, Make a Release]
+- Home: index.md
+- Installation: install.md
+- Configuration: config.md
+- Contributing: contributing.md
+- Translations: i18n.md
+- Make a Release: release_process.md
```

Comment:
That does look nicer!

However, how am I supposed to know that this is the right syntax when reviewing this pull request?

Please add a link to the relevant documentation page in the commit message, as that's always useful in the future.